docs/security.rst

```diff
@@ -58,30 +58,183 @@ SSL
 ---
 SSL should be used when client encryption is enabled in Cassandra.
 
-Versions 3.17.0 and higher
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-To enable SSL you will need to set :attr:`.Cluster.ssl_context` to a
-``ssl.SSLContext`` instance. Optionally, you can also set :attr:`.Cluster.ssl_options`
+With version 3.17.0 and higher, you will need to set :attr:`.Cluster.ssl_context` to a
+``ssl.SSLContext`` instance to enable SSL. Optionally, you can also set :attr:`.Cluster.ssl_options`
 to a dict of options. These will be passed as kwargs to ``ssl.SSLContext.wrap_socket()``
 when new sockets are created.
 
-For example:
+The following examples assume you have generated your cassandra certificate and
+keystore files with these intructions:
+
+* `Setup SSL Cert <https://docs.datastax.com/en/dse/6.7/dse-admin/datastax_enterprise/security/secSetUpSSLCert.html>`_
+
+It might be also useful to learn about the different levels of identity verification to understand the examples:
+
+* `Using SSL in DSE drivers <https://docs.datastax.com/en/dse/6.7/dse-dev/datastax_enterprise/appDevGuide/sslDrivers.html>`_
+
+SSL Configuration Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+No identity verification
+++++++++++++++++++++++++
+
+No identity verification at all. Note that this is not recommended for for production deployments.
```

Comment:
This is a bit confusing, because setting the `require_client_auth` on the server side to disabled is default, and also that it doesn't mean the client can't verify the server.